src/main/java/net/jonathangiles/tools/teenyhttpd/json/TeenyJson.java

```diff
@@ -0,0 +1,377 @@
+package net.jonathangiles.tools.teenyhttpd.json;
+
+
+import net.jonathangiles.tools.teenyhttpd.util.ParameterType;
+import net.jonathangiles.tools.teenyhttpd.util.ReflectionUtils;
+
+import java.beans.Transient;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.lang.reflect.Parameter;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.*;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+/**
+ * 0 dependencies Teeny JSON serializer
```

Comment:
Would it be worth putting TeenyJson into a separate PR once Winter is reviewed and merged, or is it used as part of the Winter feature?

Reply:
I Agree, TeenyJson should be in a separate PR (I even thought about it before coding winter) , and Winter should be an implementation detail